lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -23,52 +23,35 @@ class TraceMiddleware
         end
         option :distributed_tracing, default: false
 
-        def initialize(app, options = {})
```

Comment:
it's another breaking change, but it's better to crash applications immediately at start-up time rather than at runtime. This will simplify the migration.